src/ValidatorKeys.cpp

```diff
@@ -94,41 +108,40 @@ ValidatorKeys::make_ValidatorKeys (
             jKeys["key_type"].toStyledString());
     }
 
-    auto const masterSecret = parseBase58<SecretKey> (
-        TokenType::TOKEN_NODE_PRIVATE, jKeys["master_secret_key"].asString());
+    auto const secret = parseBase58<SecretKey> (
+        TokenType::TOKEN_NODE_PRIVATE, jKeys["secret_key"].asString());
 
-    if (! masterSecret)
+    if (! secret)
     {
         throw std::runtime_error (
             "Key file '" + keyFile.string() +
-            "' contains invalid master secret: " +
-            jKeys["master_secret_key"].toStyledString());
+            "' contains invalid secret: " +
+            jKeys["secret_key"].toStyledString());
     }
 
-    auto const masterPublicKey = derivePublicKey(keyType, *masterSecret);
+    std::uint32_t tokenSequence;
+    try {
+        if (! jKeys["token_sequence"].isIntegral())
+            throw std::runtime_error ("");
 
-    if (! jKeys["sequence"].isIntegral())
+        tokenSequence = jKeys["token_sequence"].asUInt();
```

Comment:
That's tested here:
https://github.com/wilsonianb/validator-keys-tool/blob/089e660790c8c67e194fae695ce0378ccb8160d0/src/test/ValidatorKeys_test.cpp#L153